lib/chat_app_web/live/home_live.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ChatAppWeb.HomeLive do
+  use ChatAppWeb, :live_view
+
+  def mount(_params, _session, socket) do
+    rooms = [%{id: 1, name: "room1", description: "capybara numero uno"}, %{id: 2, name: "room2", description: "capybara numero dos"}, %{id: 3, name: "room3", description: "capybara numero tres"}]
+    socket = assign(socket, :rooms, rooms)
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~H"""
+    <.header class="hd">Public Rooms</.header>
+    <.button id="newroom" class="hd">New Room</.button>
+
+    <.table id="rooms" rows={@rooms}>
```

Comment:
aici @rooms nu trebuie luat din assigns?

Reply:
@danfilip acel "@" in HEEX templates presupune ca variabila rooms e luata din assigns.